ui/browser/views/navbar/navbar.jsx

```diff
@@ -26,65 +26,76 @@ class NavBar extends Component {
     } = this.props;
 
     if (page == null) {
-      return <div id="browser-navbar"></div>;
+      return <div id='browser-navbar'></div>;
     }
 
     return (
-      <div id="browser-navbar">
-        <Btn title="Menu" image="glyph-menu-16.svg"
-          onClick={openMenu} />
+      <div id='browser-navbar'>
+        <Btn id='browser-navbar-menu'
+          title='Menu' image='glyph-menu-16.svg'
+          clickHandler={openMenu} />
 
-        <a id="pages-button" style={{
-          borderWidth: "0px 1px",
-          borderColor: "#D6D6D6",
-          borderStyle: "solid",
-          margin: "8px 8px 0px 8px",
-          padding: "0px 10px",
-          height: "22px",
+        <a id='pages-button' style={{
```

Comment:
nor here or anywhere else for attributes.
we should make eslint complain about this stuff if it isn't already.